public/static/docs/command-reference/gc.md

```diff
@@ -78,11 +87,12 @@ $ du -sh .dvc/cache/
 7.4G    .dvc/cache/
 ```
 
-When you run `dvc gc` it removes all objects from cache that are not referenced
-in the <abbr>workspace</abbr> (by collecting hash values from the DVC-files):
+When you run `dvc gc --workspace` it removes all objects from cache that are not
```

Comment:
Minor: I'd say let's use the shorthand -w for the example?
```suggestion
When you run `dvc gc -w`, DVC removes all objects from cache that are not
```

Reply:
I don't know what's more used in the docs, but, I'd say we should prefer long form flags (they are pretty self-explanatory) wherever possible. Or, unless we want to show combinations of flag like following:
```sh
$ dvc gc -caT  # 🐈
```

I'd leave to the user to use/find short-form flags.

Reply:
No strong opinion, up to you.

Reply:
But lets change "it" for "DVC" at least 🙂